src/transports/HTTPTransport.ts

```diff
@@ -4,14 +4,16 @@ import ITransport from "./Transport";
 class HTTPTransport implements ITransport {
   private uri: string;
   private onDataCallbacks: any[];
```

Comment:
if data callbacks has a type then you can use type inference I think later on when iterating through the callbacks

Reply:
or nit. might be to just pull the call back signature into a type because its so widely used. it'll maybe make the code easier to parse